<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckerboardDetectionParametersControlWidget</class>
 <widget class="QWidget" name="CheckerboardDetectionParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckerboardDetectionParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Estimate undistorted from distorted</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="estimateUndistortedFromDistortedCheckBox">
       <property name="toolTip">
         <string>Estimate undistorted from distorted</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Use Undistortion</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="useUndistortionCheckBox">
       <property name="toolTip">
         <string>Use Undistortion</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Algorithm</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="algorithmComboBox">
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Homebrew Detector</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenCV Detector</string>
        </property>
       </item>
       <property name="toolTip">
         <string>Algorithm</string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>Channel</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="channelComboBox">
       <property name="currentIndex">
         <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>R</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_red.png</normaloff>:/new/colors/colors/color_red.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_green.png</normaloff>:/new/colors/colors/color_green.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_blue.png</normaloff>:/new/colors/colors/color_blue.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_gray.png</normaloff>:/new/colors/colors/color_gray.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_gray.png</normaloff>:/new/colors/colors/color_gray.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hue</string>
        </property>
        <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/new/colors/colors/color_wheel.png</normaloff>:/new/colors/colors/color_wheel.png
        </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Saturation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Value</string>
        </property>
       </item>
       <property name="toolTip">
         <string>Channel</string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>Cell Size Hor</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="cellSizeHorSpinBox">
       <property name="toolTip">
         <string>Cell Size Hor</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>Cell Size Vert</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="cellSizeVertSpinBox">
       <property name="toolTip">
         <string>Cell Size Vert</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>Clean existing</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="cleanExistingCheckBox">
       <property name="toolTip">
         <string>Clean existing</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>Precise Diameter</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="preciseDiameterSpinBox">
       <property name="toolTip">
         <string>Precise Diameter</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>Iteration count</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="iterationCountSpinBox">
       <property name="toolTip">
         <string>Iteration count</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
       <property name="text">
         <string>Min accuracy</string>
       </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="minAccuracySpinBox">
       <property name="toolTip">
         <string>Min accuracy</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>999999</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>0.01</double>
       </property>
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
       <property name="text">
         <string>Partial Board</string>
       </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="partialBoardCheckBox">
       <property name="toolTip">
         <string>Partial Board</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
       <property name="text">
         <string>Fast board Speedup</string>
       </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="fastBoardSpeedupCheckBox">
       <property name="toolTip">
         <string>Fast board Speedup</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
       <property name="text">
         <string>Draw SGFs on boards</string>
       </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QCheckBox" name="drawSGFsOnBoardsCheckBox">
       <property name="toolTip">
         <string>Draw SGFs on boards</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_13">
       <property name="text">
         <string>Skip undistorted with no distorted board</string>
       </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QCheckBox" name="skipUndistortedWithNoDistortedBoardCheckBox">
       <property name="toolTip">
         <string>Skip undistorted with no distorted board</string>
       </property>
     </widget>
    </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>estimateUndistortedFromDistortedCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>useUndistortionCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>algorithmComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>channelComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>cellSizeHorSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>cellSizeVertSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>cleanExistingCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>preciseDiameterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>iterationCountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>minAccuracySpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>partialBoardCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>fastBoardSpeedupCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>drawSGFsOnBoardsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>skipUndistortedWithNoDistortedBoardCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CheckerboardDetectionParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
